File: Facade Pattern C# version/FacadeConsole/CdPlayer.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace FacadeConsole
{
    public class CdPlayer
    {
        public bool active = false;

        public void TurnOn()
        {
            active = true;
        }

        public void TurnOff()
        {
            active = false;
        }

        public void Eject()
        {
            Console.WriteLine("Ejecting Cd.\n");
        }

        public void Pause()
        {
            Console.WriteLine("You have paused the Cd.\n");
        }

        public void Play(String cd)
        {
            Console.WriteLine("Playing " + cd + ".\n");
        }

        public void Stop()
        {
            Console.WriteLine("You have stopped the Cd.\n");
        }
    }
}

File: Facade-Testing/src/CdPlayer.java
public class CdPlayer {
    boolean on = false;

    public void on(){
        this.on = true;
    }

    public void off(){
        this.on = false;
    }

    public void eject(){
        System.out.println("Ejecting Cd.\n");
    }

    public void pause(){
        System.out.println("You have paused the Cd.\n");
    }

    public void play(String cd){
        System.out.println("Playing " + cd + ".\n");
    }

    public void stop(){
        System.out.println("You have stopped the Cd.\n");
    }
}
